node/cli/src/lib.rs

```diff
@@ -71,13 +81,13 @@ pub fn run() {
                 .help("Sets a base path for persisted files")
                 .takes_value(true),
         ).arg(
-            Arg::with_name("chain_spec_file")
-                .short("c")
-                .long("chain-spec-file")
-                .value_name("CHAIN_SPEC_FILE")
-                .help("Sets a file location to read a custom chain spec")
-                .takes_value(true),
-        ).get_matches();
+        Arg::with_name("chain_spec_file")
```

Comment:
No indentation here?

Reply:
meh probably should have indent.. changing